File: Signiture-Backend/delivery/api/src/main/java/pcy/study/api/config/swagger/SwaggerConfig.java
package pcy.study.api.config.swagger;

import com.fasterxml.jackson.databind.ObjectMapper;
import io.swagger.v3.core.jackson.ModelResolver;
import io.swagger.v3.oas.annotations.OpenAPIDefinition;
import io.swagger.v3.oas.annotations.info.Info;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

@OpenAPIDefinition(
        info = @Info(
                title = "Delivery Service API 명세서",
                description = "배달 플랫폼 API 명세서"
        )
)
@Configuration
public class SwaggerConfig {

    @Bean
    public ModelResolver modelResolver(ObjectMapper objectMapper) {
        return new ModelResolver(objectMapper);
    }
}

File: Signiture-Backend/kotlin/delivery/api/src/main/kotlin/pcy/study/api/config/swagger/SwaggerConfig.kt
package pcy.study.api.config.swagger

import com.fasterxml.jackson.databind.ObjectMapper
import io.swagger.v3.core.jackson.ModelResolver
import io.swagger.v3.oas.annotations.OpenAPIDefinition
import io.swagger.v3.oas.annotations.info.Info
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration

@OpenAPIDefinition(
    info = Info(
        title = "Delivery Service API 명세서",
        description = "배달 플랫폼 API 명세서"
    )
)
@Configuration
class SwaggerConfig {

    @Bean
    fun modelResolver(
        objectMapper: ObjectMapper
    ): ModelResolver {
        return ModelResolver(objectMapper)
    }
}
